nmdc_server/config.py

```diff
@@ -62,6 +62,9 @@ class Settings(BaseSettings):
     rancher_backend_workload_id: Optional[str] = None
     rancher_worker_workload_id: Optional[str] = None
 
+    # CORS settings necessary for allowing request from Field Notes app
+    cors_allow_origins: Optional[str] = None  # comma separated list of allowed origins
```

Comment:
Assuming the comma-delimited string can be defined in an `.env` file, I'd recommend adding an example to [.env.example](https://github.com/microbiomedata/nmdc-server/blob/main/.env.example) because I think people might not realize the string can be defined in that file (as opposed to editing this Python script).